edx/edx-enterprise
# [ENT-416] Data sharing consent - UI issue on mobile view

**Description:** This PR fixes the layout issue of the grant data sharing modal when the screen is narrow. The modal content is also scrollable now.

**JIRA:** [ENT-416](https://openedx.atlassian.net/browse/ENT-416)

**Screenshots:**
![narrow](https://user-images.githubusercontent.com/560781/26954724-ea9af144-4c6e-11e7-90ac-1de077a06402.png)
![scroll](https://user-images.githubusercontent.com/560781/26954725-edf3e1e8-4c6e-11e7-8393-21c3bf0b45f0.png)

**Dependencies:** None 

**Merge deadline:** None

**Installation instructions:** Standard enterprise installation.

**Testing instructions:**

1. Set up a devstack with this branch.
2. Add an Enterprise Customer and enable data sharing consent at enrollment.
3. Add a learner to the Enterprise Customer.
4. Log in as the learner to get the grant_data_sharing_permissions view.
5. Click the "No, take me back." link to display the popup.
6. Make the browser window very narrow and verify that its layout doesn't break.
7. To test the scrolling of the modal you can either modify the `enterprise/templates/enterprise/grant_data_sharing_permissions.html` template in the branch or use your browser developer tools to add content to the `.modal-content div.scrollable` element.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.~

**Post merge:**



**Reviewers**
